repl/go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/google/cel-go/repl
+
+go 1.15
+
+require (
+	github.com/chzyer/readline v1.5.0
+	github.com/google/cel-go v0.11.3
+	google.golang.org/genproto v0.0.0-20220502173005-c8bf987b8c21
+)
```

Comment:
You can consider doing a local replace here as well: `replace github.com/google/cel-go => ./..`

This way you're making sure that the code is always building against the parent module.

Reply:
done